java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6804.html

```diff
@@ -0,0 +1,60 @@
+<p>This rule reports when the Spring <code>@Value</code> annotation is used to inject a simple value that does not contain an expression.</p>
+<h2>Why is this an issue?</h2>
+<p>The purpose of the <code>@Value</code> annotation in <code>org.springframework.beans.factory.annotation</code> is to inject a value into a field or
+method based on the Spring context after it has been established.</p>
+<p>If the annotation does not include an expression (either Spring Expression Language or a property injection), the injected value is a simple
+constant that does not depend on the Spring context, making the annotation replaceable with a standard field initialization statement.</p>
+<p>This not only implies the redundant use of <code>@Value</code>, but could also indicate an error where the expression indicators (<code>#</code>,
+<code>$</code>) were omitted by mistake.</p>
+<h3>Exceptions</h3>
+<p>This rule does not raise an issue if <code>@Value</code> is applied to a method or method argument, because the annotation has the side effect that
+the method is called.</p>
+<h2>How to fix it</h2>
+<ul>
+  <li> If a property is to be injected, use <code>${propertyname}</code> instead of <code>propertyname</code>. </li>
```

Comment:
Small remark of using camel case for propertyName:
If a property is to be injected, use `${propertyName}` instead of `propertyName`.

Reply:
:+1: